lib/insights/src/lib/analytics-process/components/analytics-report-parameters.component.ts

```diff
@@ -187,11 +186,11 @@ export class AnalyticsReportParametersComponent implements OnInit, OnChanges, On
     }
 
     convertMomentDate(date: string) {
```

Comment:
needs return type

Reply:
Method created by you, but return type added.

Reply:
method was created by you. But I have added the return type.